### ISSUE TITLE
Loss explodes for training mb-melgan

### ISSUE THREAD

**Issue by tts-nlp** [TensorSpeech/TensorFlowTTS#538]
loss explodes when I use multiband_melgan.baker.v1.yaml config
![Screenshot from 2021-04-08 18-08-50](https://user-images.githubusercontent.com/58066364/114009518-cb1a9c80-9895-11eb-8904-06fb714dd414.png)

**Comment by dathudeptrai** [comment]
@tts-nlp can you resume the training progress ?

**Comment by tts-nlp** [comment]
I'm training from the beginning, generator training is normal, discriminator training to 300K will gradient explosion

my config changes 
`batch_size=128`
tensorflow==2.4.1

training log is
![Screenshot from 2021-04-09 08-30-11](https://user-images.githubusercontent.com/58066364/114113359-fd1d1480-9910-11eb-9fd1-cbe96d825343.png)

**Comment by zhangsanfeng86** [comment]
reduce learning rate

**Comment by tts-nlp** [comment]
At 300K steps, the learning rate becomes half of the before, and the loss will increase later， The learning rate has been reduced. Do need to reduce it now ?

**Comment by tts-nlp** [comment]
midify batch_size=64 solve the problem， thanks !
